vcluster/_fragments/integrations/cert-manager.mdx

```diff
@@ -0,0 +1,197 @@
+import Flow, { Step } from '@site/src/components/Flow'
+import NavStep from '@site/src/components/NavStep'
+import Button from '@site/src/components/Button'
+import Label from '@site/src/components/Label'
+import Field from '@site/src/components/Field'
+
+import CertManagerPartial from '../../_partials/config/integrations/certManager.mdx';
+
+import BasePrerequisites from '../../../platform/_partials/install/base-prerequisites.mdx';
+
+import CodeBlock from '@theme/CodeBlock';
+
+import Deploy from '../../_partials/deploy/deploy.mdx'
+
+import ProAdmonition from '../../_partials/admonitions/pro-admonition.mdx'
+
+<ProAdmonition/>
+
+### Prerequisites
+
+<BasePrerequisites />
+
+- `cert-manager` operator installed on your host cluster. See instructions at https://cert-manager.io
+
+## Cert manager integration
+
+To enable the cert-manager integration, set the following fields as shown below:
+
+
+```yaml title="Enable cert-manager integration"
+integrations:
+  certManager:
+    enabled: true
+```
+
+This configuration enables the integration, imports cluster-scoped ClusterIssuers from the host cluster into the virtual cluster, and exports namespaced Issuers and Certificates from the virtual cluster into the host cluster.
+
+<Flow id="cert-manager-integration">
+ <Step>
+
+  Create the Issuer
+
+Create a <Label>file</Label> named `issuer.yaml` with the following content:
+
+```yaml title="ClusterIssuer configuration"
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    # highlight-start
+    # Replace this email address with your own.
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    # highlight-end
+    email: user@example.com
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource that will be used to store the account's private key.
+      name: example-issuer-account-key
+    # Add a single challenge solver, HTTP01 using nginx
+    solvers:
+    - http01:
+        ingress:
+          ingressClassName: nginx
+```
+
+</Step>
+<Step>
+Apply the Issuer configuration:
+
+```bash title="Apply the Issuer configuration"
+kubectl apply -f issuer.yaml
+```
+:::note
+This creates a corresponding Issuer in the host cluster.
+:::
+
+</Step>
+<Step>
+
+Create and apply Issuer and Certificate
+
+After the virtual cluster is running, create an Issuer and Certificate inside the virtual cluster. This guide uses a `letsencrypt-staging` issuer for demonstration purposes.
```

Comment:
There's only an example of applying the cert, but not an issuer in the virtual cluster.

Either we ned to update what we're going to do or give an example of how to create the issuer.